save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/agent/SaveCliOverrides.kt

```diff
@@ -0,0 +1,18 @@
+package com.saveourtool.save.agent
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property batchSize corresponds to flag `--batch-size` of save-cli (optional)
+ * @property batchSeparator corresponds to flag `--batch-separator` of save-cli (optional)
+ * @property overrideExecCmd corresponds to flag `--override-exec-cmd` of save-cli (optional)
+ * @property overrideExecFlags corresponds to flag `--override-exec-flags` of save-cli (optional)
+ */
+// FIXME: need to replace with [com.saveourtool.save.core.plugin.PluginConfig]
+@Serializable
+data class SaveCliOverrides(
+    val batchSize: Int? = null,
+    val batchSeparator: String? = null,
```

Comment:
should be nullable with null default values?

Reply:
moved current logic, probably it can be without default value, but defiantly it should be nullable